src/containers/Routes.tsx

```diff
@@ -22,8 +22,8 @@ function MainRoutes() {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/register-organization" element={<OrgRegister />} />
-          <Route path="/org-login" element={<ProtectedRoutes ><Orglogin /></ProtectedRoutes>}  />
-          <Route path="/login-admin" element={<ProtectedRoutes ><Adminlogin /></ProtectedRoutes>} />
+          <Route path="/org-login" element={<Orglogin />} />
+          <Route path="/signin" element={<Adminlogin />} />
           <Route path="/pricing" element={<Pricing />} />
           <Route path="/noredirect" element={<Noredirect />} />
```

Comment:
Why do we need a route to `noredirect` actually. I thought it was just a placeholder component

Check it out